ydb/library/yql/providers/yt/lib/res_pull/res_or_pull.cpp

```diff
@@ -3,6 +3,8 @@
 #include <ydb/library/yql/providers/common/codec/yql_codec.h>
 #include <ydb/library/yql/utils/yql_panic.h>
 
+#include <ydb/library/yql/utils/log/log.h>
```

Comment:
Do we need this?

Reply:
No, it was for the debug purposes. Will remove.